Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -171,16 +169,16 @@ ui close.
 ```
 
 
-Note that to update the contents of your window once it is open, you have the method `SpPresenter>>withWindowDo:`.
-But we will discuss it later in this book. More information about managing windows: e.g., opening dialog boxes or setting the about text is present in Chapter *@cha_managing_windows@*.
+Note that to update the contents of your window once it is open, you have the method `SpPresenter>>withWindowDo:`,
+but we will discuss it later in this book. More information about managing windows: e.g., opening dialog boxes or setting the about text is present in Chapter *@cha_managing_windows@*.
 
-This concludes our first example of a Spec user interface. We now continue with more examples on how to configure the different widgets that can be used in such a user interface.
+This concludes our first example of a Spec user interface. In the next chapter we continue with more examples on how to configure the different widgets that can be used in such a user interface.
 
 ### Conclusion
 
 
 In this chapter we have given you a little example of Spec user interfaces. We have first shown you what the different steps are to build a user interface with Spec.
 
-More examples of Spec user interfaces are found in the Pharo Image itself. Since all Spec user interfaces are subclasses of `SpPresenter`, they are easy to find and each of them may serve as an example. Furthermore, experimentation with widgets and user interfaces is made easy because all widgets can be opened as standalone windows.
+More examples of Spec user interfaces are found in the Pharo image itself. Since all Spec user interfaces are subclasses of `SpPresenter`, they are easy to find and each of them may serve as an example. Furthermore, experimentation with widgets and user interfaces is made easy because all widgets can be opened as standalone windows.
```

Comment:
I am not sure about this change. Is "Pharo Image" the way to spell it? I didn't think so, so I changed it, but tell me what you think.

Reply:
image is better. I do not why we got a Image in the first step.